Meeting notes 18th Feb 2021 (#112)
umbraco/Umbraco.Packages: team-meeting-notes/2021-02-18.md

# Package team meeting 18th Feb 2021

Attendees: 

> 	Jesper, Kevin, Lottie, Nik, Andy, David

Reviewer: ```suggestion
	Jesper, Kevin, Lotte, Nik, Andy, David & Richard
```

Notetaker:  Kevin

Agenda
 - Status on Umbpack
 - Status on Package Templates
 - Status on UI Examples
 - Status on Our
 - Any other business

## Status on Umbpack

>  - Outstanding PR (by jesper)- requires update to our (we need to confirm how/when this will be done).

Reviewer: ```suggestion
 - [Outstanding PR](https://github.com/umbraco/UmbPack/pull/59) (by jesper)- requires update to our (we need to confirm how/when this will be done).
```

 - guide/readme needs to be updated to say how to work with umbpack against local (running e.g debug/localhost)

## Status on Package Templates
 - Remove ExamplesUI templates. (Kevin)

>  - There might be some possible streamlining of template commands. (Lottie)

Reviewer: ```suggestion
 - There might be some possible streamlining of template commands. (Lotte)
```

## Status on Umbraco.Packages issues

>  

Reviewer: ```suggestion
 Nothing new to report
 
```

## Status on UI Examples
 - We have a number of bits that could be updated

>     - overlay examples.

Reviewer: ```suggestion
    - Overlay examples
```

>     - inifiete editor

Reviewer: ```suggestion
    - Infinite editor
```

> - we need to update the issues on the repo to make it easier for people to see how they can help

Reviewer: ```suggestion
- We need to update the issues on the repo to make it easier for people to see how they can help
```

>     - Move any examplesUI issues, to ther other repo. (Kevin)

Reviewer: ```suggestion
    - Move any UI Examples issues, to the other repo (Kevin)
```

> - the repo need making public, and deployable

Reviewer: ```suggestion
- The repo need making public, and deployable
```

    - Merge both PR (Kevin)

>     - request to make repo public (Jesper/Andy)

Reviewer: ```suggestion
    - Request to make repo public (Jesper/Andy)
```

>     - new API key so we can deploy. (Jesper/Andy)

Reviewer: ```suggestion
    - New API key so we can deploy (Jesper/Andy)
```
